client/Pages/EventTypeDetails/View.elm

```diff
@@ -600,32 +592,6 @@ renderPartition totalsStore eventType partition =
         ]
 
 
-publisherTab : EventType -> Stores.Publisher.Model -> String -> String -> Html Msg
-publisherTab eventType publishersStore appsInfoUrl usersInfoUrl =
-    let
-        publishersList =
-            Store.items publishersStore
-
-        count =
-            List.length publishersList
-
-        countStr =
-            pluralCount count "Publisher"
-    in
-    div [ class "dc-card" ]
-        [ span []
-            [ text countStr
-            , helpIcon "Publishers" Help.publishers BottomRight
-            , refreshButton LoadPublishers
-            ]
-        , div [ class "publisher-tab__list" ]
-            [ Helpers.Panel.loadingStatus publishersStore <|
-                grid [ "Publisher application", "Http posts in 4 days", "" ]
-                    (publishersList |> List.map (renderPublishers eventType.name appsInfoUrl usersInfoUrl))
-            ]
-        ]
-
-
 renderPublishers : String -> String -> String -> Stores.Publisher.Publisher -> Html Msg
```

Comment:
Probably we can also remove `renderPublishers`

Reply:
Done in f29013e5d672938e71da485064c67ad8aa9c57a0